src/app/components/forms/multi-check-box/terra-multi-check-box.component.ts

```diff
@@ -58,6 +62,8 @@ export class TerraMultiCheckBoxComponent implements OnInit, OnDestroy, ControlVa
 
     protected headerCheckboxValue:boolean;
 
+    protected collapsed:boolean = true;
```

Comment:
maybe use as @Input() to let the user decides when to be collapsed? Default must be false by the way... because all views with multi-checkboxes will be different than now

Reply:
I agree with the default value. But we should not make this an `@Input()`, because this will lead to different views (some want it collapsed by default and some not).

Reply:
Resolved in private discussions